apikit-tools/apikit-scaffolder/src/main/java/org/mule/tools/apikit/output/MuleConfigGenerator.java

```diff
@@ -56,8 +55,8 @@
             "http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd"
     );
     public static final NamespaceWithLocation HTTPN_NAMESPACE = new NamespaceWithLocation(
```

Comment:
We should not use this declaration anymore. We already have a declaration named HTTP_NAMESPACE